e2e/tests/ui-driven/src/helpers/context.ts

```diff
@@ -65,23 +67,24 @@ export async function setUpTestContext(
     });
   }
   if (
-    context.flow?.slug &&
-    context.flow?.data &&
-    context.flow?.name &&
+    context.flows &&
+    context.flows[0].slug &&
```

Comment:
suggestion: to make this easier to read you could make a const at the top called `firstFlow` or something like that and then use that everywhere that `context.flows[0]` is used. 

E.g.
```
const firstFlow = context.flows[0]
...
firstFlow.slug &&
firstFlow.data &&
...
```

You can probably think of a better name!